pyresample/geometry.py

```diff
@@ -1360,6 +1356,9 @@ def get_area_slices(self, area_to_cover):
 
             return slice(xstart, xstop), slice(ystart, ystop)
 
+        if self.proj_dict.get('proj') != 'geos':
+            raise NotImplementedError('Only geos supported')
```

Comment:
We could maybe just elaborate the error message a bit, because we do now support other projections than `geos` after this PR.

Reply:
Sounds good. I'll adjust it later. I suppose we still don't want to allow non-geos projections here? Other projections *should* work but iirc you said there were some that wouldn't work and would return weird results.

I am OK with that happening. I understand if you are not.